torch/csrc/jit/codegen/cuda/codegen.cpp

```diff
@@ -719,7 +719,7 @@ class CudaKernelGenerator : private OptOutConstDispatch {
         indent() << "if (rng_subseq != subseq" << uop->name()
                  << " || rng_offset != offset" << uop->name() << ") {\n";
         indent() << "  rng_result = philox(philox_args.seed_, subseq"
-                 << uop->name() << ", offset / 4 + offset" << uop->name()
+                 << uop->name() << ", philox_offset / 4 + offset" << uop->name()
```

Comment:
Do we need to rename the `offset` here too?

Reply:
I see. This one is suffixed by the uop name so would be ok.

Reply:
The second offset is `offset123`, so not necessary. But it sounds like a good idea to rename it as well. Will push an update.